lib/tasks/sap_update_authorizations.rake

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+namespace :sap do
+  desc "Updates users metadata in authorizations by SAP"
+  task update_metadata: :environment do
+    Decidim::User.find_each do |user|
+      authorization = Decidim::AuthorizationHandler.handler_for(
+        "sap_authorization_handler",
+        user: user
+      )
+      Decidim::CreateSapAuthorization.call(authorization) do
+        on(:ok) do
+          Rails.logger.info "UpdateMetadata: --- INFO: USER with #id: #{user.id} doesn't have any metadata." if authorization.metadata.values.compact.empty?
```

Comment:
I'm not sure if we should do a `reload` here or not... depending on if the command is reusing the same `authorization` obejct or not
```suggestion
          Rails.logger.info "UpdateMetadata: --- INFO: USER with #id: #{user.id} doesn't have any metadata." if authorization.reload.metadata.values.compact.empty?
```